CHANGELOG.md

```diff
@@ -6,6 +6,32 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+- CUMULUS-1171
+  - Added `@cumulus/common` API documentation to `packages/common/docs/API.md`
+  - Added an `npm run build-docs` task to `@cumulus/common`
+  - Added `@cumulus/common/string#isValidHostname()`
+  - Added `@cumulus/common/string#match()`
+  - Added `@cumulus/common/string#matches()`
+  - Added `@cumulus/common/string#toLower()`
+  - Added `@cumulus/common/string#toUpper()`
+  - Added `@cumulus/common/URLUtils#buildURL()`
+  - Added `@cumulus/common/util#isNil()`
+  - Added `@cumulus/common/util#isNull()`
+  - Added `@cumulus/common/util#isUndefined()`
+  - Added `@cumulus/common/util#negate()`
+
+### Changed
+- CUMULUS-1171
```

Comment:
This needs to be called out as a breaking change with steps to migrate. i.e. update all providers with all required fields. What will happen if they have an existing provider that does not have all the fields and they run a workflow? Will they get an error that explains what the issue is?

Reply:
As of 03f9fcd19, trying to discover or download using an invalid `provider.host` will throw a `TypeError` with a descriptive message.